### ISSUE TITLE
"Error : Ancient block sync is still in progress" when running parity with explorer

### ISSUE THREAD

**Issue by ajit-soman** [paritytech/parity-ethereum#10569]
_Before filing a new issue, please **provide the following information**._

- **Parity Ethereum version**: 2.3.8
- **Operating system**: Linux
- **Installation**: one-line installer
- **Fully synchronized**: yes
- **Network**: Private POA blockchain
- **Restarted**: no

I am using this explorer : https://github.com/carsenk/explorer to create explorer for my private parity blockchain. but i am getting ` Ancient block sync is still in progress` error . 
![Screenshot from 2019-04-06 17-21-27](https://user-images.githubusercontent.com/39642842/55669100-0b06ad80-5891-11e9-958d-bda25c121975.png)

This is my parity toml file

```
[parity]
chain = "parity-genesis.json"
[network]
port = 30300
[rpc]
port = 8000
apis = ["web3", "eth", "net", "personal", "parity", "parity_set", "traces", "rpc", "parity_accounts"]
cors = ["http://localhost:8545"]
[account]
password = ["parity-node.pass"]
unlock = ["0x289c2b76e329f962d26a9a8d9692b2f7db3bf8e3"]
[mining]
engine_signer = "0xea01694f217b0d67855646b6721682fb95d56059"
min_gas_price = 20000000000
reseal_on_txs = "none"
```

**Comment by joshua-mir** [comment]
https://wiki.parity.io/JSONRPC-parity-module#parity_chainstatus allows you to verify whether ancient block sync is ongoing, you will not see that error anymore once it is complete

**Comment by imjayson** [comment]
I'm also facing this issue.
**Version:** 2.4.5
**State DB configuration:** archive +Trace

**PoA chain steps/behavior:**
Kill chain db
On startup, node syncs only the latest blocks that are mined. 
There is no console output that it is trying to download ancient blocks 
Number of blocks on the chain is only < 2000

Tried to debug using the link provided by @joshua-mir . Does `blockGap: null` means that it is still trying to download the ancient blocks or has it stalled?

Thanks.

> curl --data '{"method":"parity_chainStatus","params":[],"id":1,"jsonrpc":"2.0"}' -H "Content-Type: application/json" -X POST localhost:8545
{"jsonrpc":"2.0","result":{"blockGap":null},"id":1}

> curl --data '{"method":"eth_getTransactionByBlockNumberAndIndex","params":["0x599", "0x0"],"id":1,"jsonrpc":"2.0"}' -H "Content-Type: application/json" -X POST localhost:8545
{"jsonrpc":"2.0","error":{"code":-32000,"message":"Ancient block sync is still in progress"},"id":1}

**Comment by joshua-mir** [comment]
@imjayson if you are running an archive sync, there is no ancient block sync as you sync from the beginning of the chain, and you shouldn't be seeing that error. Can you share exactly how you are running parity and your config?

**Comment by joshua-mir** [comment]
Also, this has already been resolved and should be fixed next release https://github.com/paritytech/parity-ethereum/issues/10261

**Comment by imjayson** [comment]
@joshua-mir you are right, i checked and there wasn't any transaction in the block i'm querying.
it's just a wrong error message. 

thank you so much!
